src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Generated Storage management plane Track1 SDK in PowerShell codebase and changed code references to it 
```

Comment:
Might should change "SDK" to "SDK code", since not include SDK in it.
And track1 is internal word, should not be public.

How about:
Replaced "Microsoft.Azure.Management.Storage" SDK reference to the generated SDK code.

Reply:
Yeah this one sounds good. Just updated the phrasing.